Demo/Sources/Demo/DemoViews/ComponentDemoViews.swift

```diff
@@ -69,6 +69,9 @@ public enum ComponentDemoViews: String, DemoViews {
     case blockUserView
     case loadingRetryView
     case promotionView
+    case recentlyfavorited
```

Comment:
maybe call them `recentlyFavoritedShelf` and `savedSearchShelf`, not to confuse them with other things 😄 There's a lot of "search" stuff in FINN 😂